include/arch/arc/v2/linker.ld

```diff
@@ -134,7 +134,21 @@ SECTIONS {
 	_image_rom_size = _image_rom_end - _image_rom_start;
 
 	GROUP_END(ROMABLE_REGION)
-
+	SECTION_PROLOGUE(.sbss,,)
```

Comment:
Unless I'm missing something, this section won't get zeroed at boot.
Can we stuff these in the regular .bss section?
Also typically these small data/bss sections are separated out like this because they are accessed in terms of a global pointer, due to being referenced by arch-specific instructions which have limited range of immediate values. See nios II's linker.ld.
I'd like to see some kind of test code prove that access to these actually works.

Reply:
small data/bss are not used or supported currently. ARC files are compiled with '-mno-sdata', there should be at most empty .sdata/.sbss sections. What problem is this PR solving?

Reply:
@ruuddw with latest gcc we are getting:

```
/home/nashif/Work/SDK/zephyr-sdk-ng-v0.1-beta2/arc-zephyr-elf/bin/../lib/gcc/arc-zephyr-elf/8.2.0/../../../../arc-zephyr-elf/bin/ld: error: linker symbol __SDATA_BEGIN__ not found
/home/nashif/Work/SDK/zephyr-sdk-ng-v0.1-beta2/arc-zephyr-elf/bin/../lib/gcc/arc-zephyr-elf/8.2.0/../../../../arc-zephyr-elf/bin/ld: final link failed: bad value
collect2: error: ld returned 1 exit status
make[3]: *** [zephyr/CMakeFiles/zephyr_prebuilt.dir/build.make:95: zephyr/zephyr_prebuilt.elf] Error 1
```